README.md

```diff
@@ -34,7 +34,10 @@ To make it easier for developers to write record processors in other languages,
 
 ## Release Notes
 
-### Latest Release (1.14.3 - May 3, 2021)
+### Latest Release (1.14.4 - June 11, 2021)
+* [#816](https://github.com/awslabs/amazon-kinesis-client/pull/816) Updated the Worker shutdown logic to make sure that the `LeaseCleanupManager` also terminates all the threads that it has started.
```

Comment:
Did we get approval for these release notes from doc writer?

Reply:
Yes.